tests/integration/test_ec2.py

```diff
@@ -0,0 +1,52 @@
+import unittest
+from localstack.utils.aws import aws_stack
+
+IMAGE_ID = 'ami-e4c81199'
+DUMMY_INSTANCE_ID = 'ABC'
+GROUP_ID = 'sg-1a2b3c4d'
+
+
+class Ec2Test(unittest.TestCase):
+
+    def setUp(self):
+        self.ec2_client = aws_stack.connect_to_service('ec2')
+
+    def test_run_instances(self):
+        result_instance = self.ec2_client.run_instances(ImageId=IMAGE_ID, InstanceType='t2.micro', MaxCount=1,
+                                                        MinCount=1)
+        actual_instance_id = result_instance['Instances'][0]['InstanceId']
+        self.assertIsNotNone(actual_instance_id)
```

Comment:
Can we extend this test and also add a call to the `describe_instance_credit_specifications` API?